package.json

```diff
@@ -2,14 +2,16 @@
   "name": "@bpmn-io/feel-editor",
   "version": "1.12.1",
   "description": "Editor for FEEL expressions.",
-  "files": [
-    "dist"
-  ],
-  "main": "dist/index.js",
-  "module": "dist/index.es.js",
+  "type": "module",
+  "main": "./dist/index.js",
+  "source": "./src/index.js",
```

Comment:
This one is non-standard, but if it's helpful to some tools, we can live with it.